pysoa/server/server.py

```diff
@@ -288,7 +288,7 @@ def run(self):
                 # Get, process, and execute the next JobRequest
                 self.handle_next_request()
         except Exception:
-            self.logger.error("Unhandled server error: %s", traceback.format_exc())
+            self.logger.error("Unhandled server error", exc_info=True)
```

Comment:
the correct way to log exceptions is to call `.exception` like `self.logger.exception("Unhandled server error")`

which _MUST_ be called within the context of an exception

Reply:
I agree with Ber, here. Use `self.logger.exception`.